### DISCUSSION TITLE
[Solved]Alerts with no data

### DISCUSSION THREAD

**Discussion by xaqpuyrthnbg** [Security-Onion-Solutions/securityonion#1800] [Unsupported Versions] [answered]
I have noticed that I will sometimes get alerts with and when I drilldown or filter it says "No data".  Have I done something to cause this or is it a bug?

Thanks, 

-D

**Comment by netsecninja** [comment]
I've seen the same behavior on 2.3.2.

**Reply by xaqpuyrthnbg** [reply]
Thanks for confirming my sanity.

**Comment by TOoSmOotH** [comment]
Can someone post a screenshot or an example?

**Reply by xaqpuyrthnbg** [reply]
It is intermittent behavior not affecting every alert.  Here is a current example of the behavior:

![Screen Shot 2020-11-03 at 7 15 31 PM](https://user-images.githubusercontent.com/40277362/98054490-0e657080-1e09-11eb-8779-e6c37b6e643a.png)
![Screen Shot 2020-11-03 at 7 14 18 PM](https://user-images.githubusercontent.com/40277362/98054484-0d344380-1e09-11eb-8f6e-ba67e41964b4.png)

**Reply by greatapoc** [reply]
I think it's an issue with the time picker, I get the same problem sometimes. I raised a similar issue earlier #1597 

Maybe try making the time window wider and see if you get data.

**Reply by xaqpuyrthnbg** [reply]
Just tried 48 and 72 hours with the same result.  Also, they were supposedly "new" alerts.  
![Screen Shot 2020-11-03 at 7 39 17 PM](https://user-images.githubusercontent.com/40277362/98055794-533ed680-1e0c-11eb-8936-c7c802ff68df.png)

**Comment by dougburks** [comment]
Please try the following:
- click the alert
- on the Quick Action Bar, click the crosshairs icon to pivot to Hunt

Does the alert display there in Hunt?

If you drill into it, does anything look strange (particularly the timestamp fields)?

**Reply by xaqpuyrthnbg** [reply]
There are alerts in Hunt, however, they are all from yesterday and these alerts should have been from an action I just took this morning. See attached screenshot:
![Screen Shot 2020-11-04 at 6 47 40 AM](https://user-images.githubusercontent.com/40277362/98108369-d8a4a400-1e69-11eb-8dcd-bb6eecc97b41.png)
![Screen Shot 2020-11-04 at 6 46 58 AM](https://user-images.githubusercontent.com/40277362/98108375-db9f9480-1e69-11eb-990d-3541311ea3e2.png)

**Reply by dougburks** [reply]
Can you confirm that the timezone on your Security Onion box is set to UTC?

What is the timezone of your local browser?

**Reply by xaqpuyrthnbg** [reply]
Running date provides this output:  Wed Nov  4 14:28:43 UTC 2020; browser is running Time Zone: America/New_York

**Reply by dougburks** [reply]
Please try the comment here:
https://github.com/Security-Onion-Solutions/securityonion/discussions/1824#discussioncomment-121636

**Reply by xaqpuyrthnbg** [reply]
Yes, that does work in Hunt.  However, in the Alerts screen when a new alert comes through and you use the drilldown button, sometimes there is no alert data for the past 24 hours.  If I look at it in Hunt, I can see that there are alerts from the last hour.  

So, I believe the data to be there, but for some reason it is not always showing up in alerts for easy review.

**Comment by xaqpuyrthnbg** [comment, marked as answer]
Good morning, 

2.3.10 seems to have resolved this issues.  I imagine the TZ DST issue is what was causing this.
